app/models/content_page.rb

```diff
@@ -5,8 +5,10 @@ class ContentPage < ApplicationRecord
   scope :top_level, -> { where("parent_id IS NULL") }
   scope :order_by_position, -> { order("position ASC") }
 
-  ONLY_ALPHA_NUMERIC_COMMA_HYPHEN_AND_SPACE = /\A[a-zA-Z0-9,.\- ]+\Z/.freeze
-  validates :title, format: { with: ONLY_ALPHA_NUMERIC_COMMA_HYPHEN_AND_SPACE, message: "should only contain alphabetic, numeric, comma, fullstop, hyphen and space characters" }
+  CHARS_TO_OMIT_FROM_SLUG = ",.()".freeze
+  ONLY_ALPHA_NUMERIC_COMMA_HYPHEN_SPACE_AND_ROUND_BRACES = /\A[a-zA-Z0-9,.\-\(\) ]+\Z/.freeze
+  TITLE_FORMAT_ERROR_MESSAGE = "should only contain alphabetic, numeric and -#{CHARS_TO_OMIT_FROM_SLUG}".freeze
+  validates :title, format: { with: ONLY_ALPHA_NUMERIC_COMMA_HYPHEN_SPACE_AND_ROUND_BRACES, message: TITLE_FORMAT_ERROR_MESSAGE }
   validates :title, presence: true, uniqueness: true
```

Comment:
Allow ( and ) in page titles

Reply:
we should really have tests for this validation